.travis.yml

```diff
@@ -13,13 +13,17 @@ python:
 install:
   - pip install codecov
   - pip install .[dev]
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then pip install -U black; fi
```

Comment:
Is there a reason that this is only explicitly for Python 3.6 and not >= 3.6?

Reply:
I think the idea was that black only needs to run once (since the formatted code is independent of the python version). But just to make sure nothing breaks once we deprecate 3.6 it is probably safer to change it to `>=3.6`. I will make that change once we have settled the above comment